__tests__/redux-mount-point.js

```diff
@@ -1,10 +0,0 @@
-import { NAME } from '../src/constants'
-
-export default NAME
-
-// dummy test
-describe('redux mount point', () => {
-  it('should be ...', () => {
-    expect(NAME).toBe(NAME)
```

Comment:
I would say we should update this to check that the "waiter" name doesn't actually change. 
`expect(NAME).toBe('waiter)`

I was using the REDUX_MOUNT_POINT because NAME isn't very clear when reading the tests. We could change the constant NAME to be REDUX_MOUNT_POINT if that's better.

Reply:
Sounds good. Will update when I get the chance.

Reply:
Test updated and changes reverted